src/Netflix/Netflix.ts

```diff
@@ -0,0 +1,760 @@
+// lipsurf-plugins/src/Netflix/Netflix.ts
+/// <reference types="lipsurf-types/extension"/>
+declare const PluginBase: IPluginBase;
+
+// Messaging
+
+const LIPSURF_BOOT_SCRIPT_ID = "lipsurf-netflix-script";
+const TO_PAGE_PROOF_KEY = String(Math.floor(Math.random() * 1000));
```

Comment:
This is cool! Any reason to not just use a static string prepended with `PluginBase.util.getNoCollisionUniqueAttr()` ?

Reply:
1.) I didn't realize it is there when I skimmed the docs and 2.) I didn't expect that function to exists in the platform utils (probably the latter influence the former).
I'll change it to that